samples/tab-personal-quickstart/js/src/components/__tests__/Tab.spec.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { act } from 'react-dom/test-utils';
+import sinon from 'sinon';
+import { app } from '@microsoft/teams-js';
+import Tab from "../Tab";
+
+sinon.stub(app, 'getContext').returns(
+  Promise.resolve({
+    user: { userPrincipalName: 'test@test.com' },
+    app: { theme: 'default' },
+  })
+);
+const registerOnThemeChangeHandlerStub = sinon
+  .stub(app, 'registerOnThemeChangeHandler')
+  .returns(Promise.resolve());
```

Comment:
NIT: [This returns a void](https://github.com/OfficeDev/microsoft-teams-library-js/blob/main/packages/teams-js/src/public/app.ts#L754)

We might want to consider Typescript if we generalize this.

Reply:
ya defintiely Typescript makes more sense